src/puyapy/__main__.py

```diff
@@ -113,12 +116,11 @@ def main() -> None:
         "--template-var",
         dest="cli_template_definitions",
         metavar="VAR=VALUE",
-        action="append",
+        action=_ParseAndStoreTemplateVar,
+        nargs="+",
         help="Define template vars for use when assembling via --output-bytecode"
         " should be specified without the prefix (see --template-vars-prefix), e.g."
-        " -T=SOME_INT=1234"
-        " -T=SOME_BYTES=0x1A2B"
-        ' -T=SOME_STR=\\"hello\\"',
+        ' -T SOME_INT=1234 SOME_BYTES=0x1A2B SOME_BOOL=True SOME_STR=\\"hello\\"',
```

Comment:
Should update docs too?

Reply:
Docs look okay to me, there's only an example value there, and technically a bool is an int too in Python